### ISSUE TITLE
issue with build map

### ISSUE THREAD

**Issue by jbanety** [jspm/jspm-cli#2468]
Sometimes when I run the command `jspm build $(jspm trace --deps ./app.js) -h -o`, the output map is wrong.

Example of output :
```json
{
  "imports": {
    "@etdsolutions/stacks": "./dist/stacks-ff183556.js",
    "@erp/ui-components/parcels/list.parcel.js": "./dist/request-db1f16d4.js",
    "@etdsolutions/request": "./dist/routie-ad357c93.js",
    "@etdsolutions/uid": "./dist/list.parcel-8b402777.js",
    "@erp/ui-components/parcels/sidenav.parcel.js": "./dist/uid-0b9cbb9d.js",
    "@etdsolutions/routie": "./dist/sidenav.parcel-466e49de.js"
  }
}
```
As we can see, the rollup generated code is not corresponding to the good bare specifier.
After investigating, it seems that jspm's `inputObj` and rollup's `output` are not is the same order but I can't understand wky.

Any idea ?

**Comment by guybedford** [comment]
Very interesting. This sounds like a Rolllup bug then as the sort is supposed to be stable.

Rollup does its output sort here - https://github.com/rollup/rollup/blob/master/src/rollup/index.ts#L359.

I guess the problem then is the outputBundle object not being populated in entry point order first.

Definitely an important bug to fix.

Alternatively I wonder if we could do some sort of matching in jspm to correct for this.

**Comment by jbanety** [comment]
I create a PR #2470 to fix this issue.

**Comment by guybedford** [comment]
Released in 2.0.0-beta.4.

**Comment by guybedford** [comment]
https://github.com/jspm/jspm-cli/pull/2480
